js/script.js

```diff
@@ -4,12 +4,25 @@
 'use strict';
 
 const FIRMWARE_API = "//io.adafruit.com"
+const QUICK_START_LINK = "https://learn.adafruit.com/quickstart-adafruit-io-wippersnapper/installing-wippersnapper"
 const DO_DOWNLOAD = false
 
+const BOARD_TO_CHIP_MAP = {
+  'feather-esp8266': ESP8266,
+  'feather-esp32': ESP32,
+  'feather-esp32-v2-daily': ESP32
+}
```

Comment:
I think you are proposing it would look like:
```javascript
const BOARD_TO_CHIP_MAP = {
  'esp8266': ESP8266,
  'ESP32': ESP32,
}
```
...and the call-site would change from:
```javascript
(BOARD_TO_CHIP_MAP[firmware.id] == chipType)
```
...to...
```javascript
(BOARD_TO_CHIP_MAP[firmware.mcuName] == chipType)
```
Am I getting that right?

🤔 

This would:
- require me to expose that value from the API (not hard)
- allow this map to grow more easily (scales better if LOTS of boards get added)
- need to ensure they're matchable, already see case sensitivity between 8266 and 32

I'm kinda "meh" on this, what do you think? Did I miss something important in this analysis?

Reply:
If you think this would be harder to maintain, we can leave it as is. My thoughts were more along the lines, we just define the chip type in the boards definition to make adding new boards easier.

Reply:
Cool makes sense. I'll leave it for now since everything works and I'm ready to go over to IO and start pulling things that direction. Once it is being served and populated from IO directly this kind of change is TRIVIAL.